recipes/args/all/conanfile.py

```diff
@@ -0,0 +1,41 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.28.0"
+
+class ArgsConan(ConanFile):
+    name = "args"
```

Comment:
`args` sounds very general.
Should we rename it to `taywee-args`?

Reply:
Perhaps yes. There is already a conflict: https://github.com/conan-io/conan-center-index/pull/3249

Reply:
Honestly, I didn't expect to have a name collision this soon 😂

Reply:
done